### PR TITLE
fix(Flex/react): add jest spy in tests

### PR DESCRIPTION
**Pull request by krishnaacc123**

### CODE REVIEW

File: src/atoms/Flex/__tests__/Flex.web.test.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ import Text from '../../Text';
 import Space from '../../Space';
 import Flex from '../index';
 
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
```
Comment (kamleshchandnani) [inline src/atoms/Flex/__tests__/Flex.web.test.js]: why do you need this?

Comment (shubhnik) [reply src/atoms/Flex/__tests__/Flex.web.test.js:10]: https://github.com/razorpay/blade/pull/81#discussion_r402307800

Comment (krishnaacc123) [reply src/atoms/Flex/__tests__/Flex.web.test.js:10]: To suppress the error message that comes in test case where we pass more than on child and try to throw error.

Comment (krishnaacc123) [reply src/atoms/Flex/__tests__/Flex.web.test.js:10]: Will move it to that test case instead of beforeAll

Comment (kamleshchandnani) [reply src/atoms/Flex/__tests__/Flex.web.test.js:10]: Yes please move it to that particular test case

**Pull request merged**